packages/@aws-cdk/cloudformation-include/test/test-templates/conditions.json

```diff
@@ -0,0 +1,48 @@
+{
+  "Conditions": {
+    "AndCond": {
+      "Fn::And": [
+        {"Condition": "AlwaysTrueCond"},
```

Comment:
I don't think this is valid CloudFormation, right...? What is the `"Condition"` key here?